apps/test/unit/storage/dataBrowser/dataVisualizer/VisualizerModalTest.js

```diff
@@ -198,4 +198,155 @@ describe('VisualizerModal', () => {
       ).to.deep.equal(['"xyz"', '"def"', '"123"', 'undefined', '123', 'true']);
     });
   });
+
+  describe('filterRecords', () => {
+    let wrapper;
+    beforeEach(() => {
+      wrapper = shallow(<VisualizerModal {...DEFAULT_PROPS} />);
+    });
+    it('works with numbers', () => {
+      let records = [
+        {id: 1, filterCol: 123, chartCol: 2},
+        {id: 2, filterCol: 456, chartCol: 3},
+        {id: 3, filterCol: 123, chartCol: 5},
+        {id: 4, filterCol: '456', chartCol: 7},
+        {id: 5, filterCol: 0, chartCol: 5}
+      ];
+      expect(
+        wrapper.instance().filterRecords(records, 'filterCol', '123')
+      ).to.deep.equal([
+        {id: 1, filterCol: 123, chartCol: 2},
+        {id: 3, filterCol: 123, chartCol: 5}
+      ]);
```

Comment:
we should be able to simplify these all of the expect statements for this to something like:

```javascript
expect(
  wrapper.instance().filterRecords(records, 'filterCol', '123')
).to.deep.equal([records[0], records[2]]);
```